app/uk/gov/hmrc/helptosavefrontend/controllers/BankAccountController.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosavefrontend.controllers
+
+import cats.instances.future._
+import com.google.inject.{Inject, Singleton}
+import play.api.i18n.MessagesApi
+import play.api.mvc.{Action, AnyContent, Result}
+import play.api.{Configuration, Environment}
+import uk.gov.hmrc.auth.core.AuthConnector
+import uk.gov.hmrc.helptosavefrontend.auth.HelpToSaveAuth
+import uk.gov.hmrc.helptosavefrontend.config.FrontendAppConfig
+import uk.gov.hmrc.helptosavefrontend.connectors.SessionCacheConnector
+import uk.gov.hmrc.helptosavefrontend.forms.BankDetails
+import uk.gov.hmrc.helptosavefrontend.metrics.Metrics
+import uk.gov.hmrc.helptosavefrontend.models.eligibility.IneligibilityReason
+import uk.gov.hmrc.helptosavefrontend.services.HelpToSaveService
+import uk.gov.hmrc.helptosavefrontend.util.Logging._
+import uk.gov.hmrc.helptosavefrontend.util._
+import uk.gov.hmrc.helptosavefrontend.views
+
+import scala.concurrent.Future
+
+@Singleton
+class BankAccountController @Inject() (val helpToSaveService:     HelpToSaveService,
+                                       val sessionCacheConnector: SessionCacheConnector,
+                                       val authConnector:         AuthConnector,
+                                       val metrics:               Metrics)(implicit override val messagesApi: MessagesApi,
+                                                                           val transformer:       NINOLogMessageTransformer,
+                                                                           val frontendAppConfig: FrontendAppConfig,
+                                                                           val config:            Configuration,
+                                                                           val env:               Environment)
+
+  extends BaseController with HelpToSaveAuth with EnrolmentCheckBehaviour with SessionBehaviour {
+
+  def getBankDetailsPage(): Action[AnyContent] = authorisedForHtsWithNINO { implicit request ⇒ implicit htsContext ⇒
```

Comment:
do we need to retrieve the NINO here? Could we get away with `authorisedForHts`?

Reply:
`authorisedForHts`  doesn't demand CL200 which is annoying , but here we expect CL200, which we can fix/refactor in `HelpToSaveAuth` later

Reply:
oh, I think that's because it's being used for the introduction pages where we don't need CL200. Since those pages are migrating to gov.uk soon, we'll be able to make authorisedForHts require CL200 after that. For now, using `authorisedForHtsWithNINO` should be fine - the extra retrieving of the NINO isn't going to break anything